test/fixture/registration/fixture.go

```diff
@@ -0,0 +1,26 @@
+package registration
+
+import (
+	"github.com/spiffe/spire/proto/common"
+)
+
+func GetRegistrationEntries() []*common.RegistrationEntry {
```

Comment:
Maybe this should go in `test/util`?

Reply:
It could also read the content from https://github.com/spiffe/spire/blob/master/test/fixture/registration/good.json